types/office-js-preview/index.d.ts

```diff
@@ -10748,7 +10748,8 @@ declare namespace Office {
          *
          * **Note**: The following clients have different behavior for saveAsync on appointments in compose mode:
          *
-         * - Mac Outlook does not support saveAsync on a meeting in compose mode. Calling saveAsync on a meeting in Mac Outlook will return an error.
+         * - Outlook for Mac does not support saveAsync on a meeting in compose mode so calling saveAsync in that scenario will return an error.
```

Comment:
Consider splitting this sentence into two for readability.
```suggestion
         * - Outlook for Mac does not support saveAsync on a meeting in compose mode. Calling saveAsync in that scenario returns an error.
```